### ISSUE TITLE
Uncaught ReferenceError: krajeeDialog is not defined

### ISSUE THREAD

**Issue by Resmedia** [kartik-v/yii2-dialog#35]
Hello! 
I have a situation, after when build was created on server, I got the following error and the termination of most plugins. I don't know where to look.

lock file was deleted and reinstalled vendor folder

https://prnt.sc/kotuaj

```
catalog:42 Uncaught ReferenceError: krajeeDialog is not defined
    at catalog:42
(anonymous) @ catalog:42
jquery.js:1580 Uncaught Error: Syntax error, unrecognized expression: #
    at Function.Sizzle.error (jquery.js:1580)
    at Sizzle.tokenize (jquery.js:2232)
    at Sizzle.select (jquery.js:2659)
    at Function.Sizzle [as find] (jquery.js:884)
    at jQuery.fn.init.find (jquery.js:2922)
    at new jQuery.fn.init (jquery.js:3032)
    at jQuery (jquery.js:98)
    at new ExportData (kv-export-data.js:41)
    at HTMLAnchorElement.<anonymous> (kv-export-data.js:147)
    at Function.each (jquery.js:362)
```

**I try to:** 

reinstall yii2-export

install yii2-dialog

restart services and clean cache

**How i understand** 

This happens that some was updated and Dialog now have another settings

**What i do to working**

1) revert my composer.lock
2) remove vendor
3) then composer install

**My composer**

```
{
    "name": "yiisoft/yii2-app-advanced",
    "description": "Yii 2 Advanced Project Template",
    "keywords": ["yii2", "framework", "advanced", "project template"],
    "homepage": "http://www.yiiframework.com/",
    "type": "project",
    "license": "BSD-3-Clause",
    "support": {
        "issues": "https://github.com/yiisoft/yii2/issues?state=open",
        "forum": "http://www.yiiframework.com/forum/",
        "wiki": "http://www.yiiframework.com/wiki/",
        "irc": "irc://irc.freenode.net/yii",
        "source": "https://github.com/yiisoft/yii2"
    },
    "repositories": [
        {
            "type": "composer",
            "url": "https://asset-packagist.org"
        }
    ],
    "minimum-stability": "dev",
    "require": {
        "php": ">=5.4.0",
        "yiisoft/yii2": "~2.0.6",
        "yiisoft/yii2-bootstrap": "~2.0.0",
        "yiisoft/yii2-swiftmailer": "^2.0@dev",
        "v0lume/yii2-meta-tags": "*",
        "himiklab/yii2-sitemap-module": "*",
        "guzzlehttp/guzzle": "^6.2@dev",
        "flow/jsonpath": "*",
        "codeception/phpbuiltinserver": "^1",
        "FortAwesome/Font-Awesome": "*",
        "kartik-v/yii2-widgets": "*",
        "2amigos/yii2-ckeditor-widget": "*",
        "sunhater/kcfinder": "*",
        "himiklab/yii2-sortable-grid-view-widget": "*",
        "zxbodya/yii2-image-attachment": "*@dev",
        "kartik-v/yii2-grid": "@dev",
        "2amigos/yii2-taggable-behavior": "~1.0",
        "zxbodya/yii2-gallery-manager": "*@dev",
        "kartik-v/yii2-date-range": "dev-master",
        "2amigos/yii2-selectize-widget": "~1.0",
        "lcobucci/jwt": "^3.2",
        "kartik-v/yii2-field-range": "*",
        "yiisoft/yii2-redis": "^2.0",
        "yiisoft/yii2-debug": "^2.0",
        "kartik-v/yii2-export": "^1.2@dev",
        "zelenin/yii2-rss": "~0.1",
        "iutbay/yii2-kcfinder": "dev-master",
        "sokolnikov911/yandex-turbo-pages": "*",
        "elasticsearch/elasticsearch": "5.0",
        "johnitvn/yii2-rbac-plus": "1.0.6",
        "kartik-v/yii2-editable": "*",
        "kartik-v/yii2-datecontrol": "dev-master",
        "yandex-money/yandex-checkout-sdk-php": "*",
        "lav45/yii2-aceeditor": "1.1.*",
    },

    "require-dev": {
        "yiisoft/yii2-gii": "~2.0.0",
        "yiisoft/yii2-faker": "~2.0.0",

        "codeception/base": "^2.2.3",
        "codeception/verify": "~0.3.1"
    },
    "config": {
        "process-timeout": 1800,
        "fxp-asset": {
            "repositories": {
                "bower-asset/eve": {
                    "type": "bower-vcs",
                    "url": "https://github.com/adobe-webplatform/eve.git"
                }
            }
        }
    },
    "extra": {
        "asset-installer-paths": {
            "npm-asset-library": "vendor/npm",
            "bower-asset-library": "vendor/bower"
        }
    }
}

```
#### Browsers

* [x] Google
* [x] Safari

#### Operating System

* [x] Linux

## Isolating the problem

* [x] The bug happens consistently across all tested browsers

**Comment by Resmedia** [comment]
How can i see here the same problem 

<img width="696" alt="screenshot at 02 23-32-13" src="https://user-images.githubusercontent.com/9871970/44960445-8c1b5200-af08-11e8-8ff4-491a2e46e4a9.png">

**Comment by kartik-v** [comment]
This will be fixed via kartik-v/yii2-export#252
